spec/menu_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+require 'menu'
+
+describe Menu do
+  subject(:menu) { Menu.new }
+  let(:items) { { :'Pizza' => 9.25, :'Chips' => 2.00 } }
```

Comment:
Does it not make more sense to use lowercase symbols and some sort of `humanize` method in a view layer?